eclair-core/src/main/scala/fr/acinq/eclair/CoinUtils.scala

```diff
@@ -17,14 +17,16 @@
 package fr.acinq.eclair
 
 import java.text.{DecimalFormat, NumberFormat}
+
```

Comment:
nit: unnecessary blank lines?

Reply:
It's the default import formatting (`ctrl+alt+o`).
One paragraph for java stdlib imports, then one paragraph for libraries imports, then one for scala stdlib imports.